.jenkins/build.sh

```diff
@@ -87,4 +87,14 @@ if [[ "$JOB_NAME" != *pynightly* ]]; then
    git clone https://github.com/pytorch/extension-ffi.git
    cd extension-ffi/script
    python build.py
+   cd -
+fi
+
+# Test no-Python build
+if [[ "$JOB_NAME" == pytorch-linux-xenial-cuda9-cudnn7-py3-build ]] || \
+   [[ "$JOB_NAME" == pytorch-linux-trusty-py3.6-gcc7.2-build ]]; then
+   echo "Building libtorchjit with NO_PYTHON"
```

Comment:
Can we change this to `libtorch`? It's not just the jit, it's also autograd, etc.